### ISSUE TITLE
Support for accents and ligatures is missing

### ISSUE THREAD

**Issue by mheiman** [paulrosen/abcjs#138]
The ABC standard specifies special encodings to use to generate accented characters, but ABCJS doesn't appear to support these. For instance, the string \AA in an ABC file should be converted to Å. A lot of international tunes have these characters.

The full set of encodings can be found here: http://abcnotation.com/wiki/abc:standard:v2.1#text_strings

Here's an example tune whose title isn't displayed properly:

```
X:1
T:\AA Jenta \aa ja
R:Hambo-polska
O:Sweden
M:3/4
L:1/8
K:G
P:A
|: "G"B2 A>B G2 | B2 A>B G2 | "D7"A2 A>G F>G | A>G F>E D2 |
  "G"B2 A>B G2 | B2 A>B G2 | "D7"A2 A>G F>G | A2 "G"G4 :|
P:B
|: "G"G>B d2 d2 | "C"c>d e>d c>B | "D7"A>B c2 c2 | "G"B>c d>c B>A |
"Em"G>A B2 B2 | "Am"A>B c>B A>G | "D7"F>G A>D E>F | A2 "G"G4 :|

```

**Comment by paulrosen** [comment]
Interesting... Most of the encodings are supported, but the version I was working from was a little old. At the bottom of the document that you linked, in the errata, there is the sentence:
```
Section 8.2 Text strings: Typos for accent mnemonics (cedilla and ring): "\,C \,c" and
"\oA \oa" corrected to "\cC \cc" and "\AA \aa", respectively, as per Section 14.1 
Supported accents & ligatures (8th Jan 2012).
```
I'll correct that in the next version (v3.3.5). In the meantime, you can use any unicode character you want, so you can express the title like this:
```
T:Å Jenta å ja
```

**Comment by paulrosen** [comment]
I also found a few more that were missing that I just added.

**Comment by mheiman** [comment]
Thanks. One _could_ use Unicode, but a lot of older ABC applications don't respond well to that, so the encodings are important.

**Comment by paulrosen** [comment]
Fixed in 4.0.0
